docs_src/platforms.md

```diff
@@ -2,16 +2,16 @@
 
 ## CPU
 
-11th Gen Intel® Core™
+11th Gen Intel® Core™ i5/i7
 
-12th Gen Intel® Core™
+12th Gen Intel® Core™ i5/i7
 
-Intel® xeon Platinum 8351N
+Intel® xeon Platinum 8351N i5/i7
```

Comment:
You can remove i5/i7 from this line and all of the GPU devices. Only the 11th & 12th gen need to specify